addons/hr_timesheet/report/timesheets_analysis_report.py

```diff
@@ -25,6 +25,14 @@ class TimesheetsAnalysisReport(models.Model):
     unit_amount = fields.Float("Time Spent", readonly=True)
     partner_id = fields.Many2one('res.partner', string="Partner", readonly=True)
     milestone_id = fields.Many2one('project.milestone', related='task_id.milestone_id')
+    message_partner_ids = fields.Many2many('res.partner', compute='_compute_message_partner_ids',
+                                           search='_search_message_partner_ids', readonly=True)
+
+    def _compute_message_partner_ids(self):
+        return self.env['account.analytic.line']._compute_message_partner_ids()
```

Comment:
It will not work, the compute in `account.analytic.line` will just alter the field in `account.analytic.line` and not the one in the report. 🙂 

(Also, we did not give any record to self)

Reply:
Ah true, good catch! I've update the code.

> (Also, we did not give any record to self)

yeah that's me being a dummy '-.-